README.md

```diff
@@ -0,0 +1,3 @@
+# odie-server #
+
+Postgres backend for [odie](https://github.com/fsmi/odie-client).
```

Comment:
Basically, the setup is:
* have postgres working.
* run fill_data.py

I'll add some instructions once there's an actual app to run (even just a
flask-restless-inferred one), but since I haven't got the cart mapping down
yet, that's taking a back seat for now.
On 23 Apr 2015 00:43, "Sebastian Ullrich" <notifications@github.com> wrote:

> In README.md
> <https://github.com/fsmi/odie-server/pull/12#discussion_r28922061>:
>
> > @@ -0,0 +1,3 @@
> > +# odie-server #
> > +
> > +Postgres backend for [odie](https://github.com/fsmi/odie-client).
>
> Add instructions for local setup?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fsmi/odie-server/pull/12/files#r28922061>.
>